foreman/data_refinery_foreman/surveyor/external_source.py

```diff
@@ -31,7 +31,7 @@ class PipelineEnums(Enum):
 
 class ProcessorPipeline(PipelineEnums):
     """Pipelines which perform some kind of processing on the data."""
-    MICRO_ARRAY_TO_PCL = "MICRO_ARRAY_TO_PCL"
+    AFFY_TO_PCL = "AFFY_TO_PCL"
```

Comment:
Yes to both questions. Once we actually start adding lots of processors the naming conventions may need to be even more specific (such as AFFY_HGU133A and AFFY_HGU133B).